scripts/chroot.sh

```diff
@@ -165,10 +165,10 @@ apt-get install -f
 apt-get -qq -y remove dconf-tools
 
 # Instalation of packages from Generator Webapp
-SCRIPT_URL=https://www.github.com/fossasia/meilix-generator/archive/master.zip
+SCRIPT_URL=https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/fossasia/meilix-generator/tree/master/scripts
```

Comment:
This url does not work.. 

```
$ wget -O- https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/fossasia/meilix-generator/tree/master/scripts 
--2018-07-09 19:38:08--  https://minhaskamal.github.io/DownGit/
Resolving minhaskamal.github.io (minhaskamal.github.io)... 185.199.109.153, 185.199.111.153, 185.199.108.153, ...
Connecting to minhaskamal.github.io (minhaskamal.github.io)|185.199.109.153|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2026 (2.0K) [text/html]
Saving to: ‘STDOUT’

-                     0%[                      ]       0  --.-KB/s             <!DOCTYPE html>
<html ng-app="downGit">
<head>
    
    <!-------------------------------- META -------------------------------->
    
    <meta charset="utf-8"/>
    <meta name="keywords" content="GitHub Directory Link, GitHub Directory, Github Link, GitHub Directory Download, GitHub, Download"/>
    <meta name="description" content="Create GitHub Directory Download Link"/>
    <meta name="author" content="Minhas Kamal"/>
    
    
    <title>DownGit</title>
    <link rel="icon" type="image/ico" href="res/images/downgit.png">
    
    
    <!-------------------------------- LINK -------------------------------->
    
    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous" />
    <link href="lib/angular-toastr(2.0.0).min.css" rel="stylesheet"/>
    
    <link href="app/site.css" rel="stylesheet" />
</head>
<body>
    
    <div ng-view class=""></div>
    
    
    <!------------------------------- SCRIPT ------------------------------>
    
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.6/angular-route.min.js"></script>
    <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
    <script src="lib/angular-toastr(2.0.0).tpls.min.js"></script>
    <script src="https://cdn.rawgit.com/Stuk/jszip/v3.0.0/dist/jszip.min.js"></script>
    <script src="lib/filesaver.min.js"></script>

    <script src="app/app.js"></script>
    <script src="app/home/home.js"></script>
    <script src="app/home/down-git.js"></script>
</body>
-                   100%[=====================>]   1.98K  --.-KB/s   in 0s     

2018-07-09 19:38:09 (12.7 MB/s) - written to stdout [2026/2026]
````

Reply:
I'm getting same error with `curl` also.
But the URL successfully downloads the script as zip if we only execute the URL.

Reply:
The zip file is built in the browser using JavaScript. Wget is not a browser and can not do that.

Reply:
I have no idea why you download scripts from meilix-generator. This seems to be also a security risk. I would only set variables as switches and have the scripts in meilix.